lib/portlayer/exec/container.go

```diff
@@ -377,15 +433,15 @@ func (c *Container) start(ctx context.Context) error {
 
 	detail, err = c.vm.WaitForKeyInExtraConfig(ctx, key)
 	if err != nil {
-		c.State = existingState
 		return fmt.Errorf("unable to wait for process launch status: %s", err.Error())
 	}
 
 	if detail != "true" {
```

Comment:
This error path should set state to Stopping - that's the expected behaviour if we fail to launch a process in the containerVM